leasing/serializers/rent.py

```diff
@@ -135,6 +135,15 @@ class Meta:
             "end_date",
         )
 
+    def to_internal_value(self, data):
+        if "amount" in data and "base_amount" not in data:
+            data["base_amount"] = data["amount"]
+
+        if "period" in data and "base_amount_period" not in data:
+            data["base_amount_period"] = data["period"]
+
+        return super(ContractRentSerializer, self).to_internal_value(data)
```

Comment:
You could just call `super()` without arguments here.

Reply:
That is true, it's fixed.